src/components/app/recentActivityRow/variantRecentActivityRow.tsx

```diff
@@ -253,12 +259,15 @@ export const VariantRecentActivityRow = function VariantRecentActivityRow({
   const hasJoinedLinkedIn = hasTakenAction(UserActionType.LINKEDIN, data?.performedUserActionTypes)
   const hasVotedInPoll = hasTakenAction(UserActionType.POLL, data?.performedUserActionTypes)
 
-  const fromStateOrEmpty = isStateAvailable
-    ? t('from', { state: userLocationDetails.administrativeAreaLevel1 })
-    : ''
-  const inStateOrEmpty = isStateAvailable
-    ? t('in', { state: userLocationDetails.administrativeAreaLevel1 })
-    : ''
+  const location =
+    countryCode === SupportedCountryCodes.EU
+      ? userLocationDetails?.countryCode
+        ? t(userLocationDetails.countryCode.toLowerCase() as EUCountryCode)
+        : ''
+      : (userLocationDetails?.administrativeAreaLevel1 ?? '')
```

Comment:
np: Can we simplify this piece o code for better readability? 

ex:

```suggestion
const euCode = userLocationDetails?.countryCode?.toLowerCase() as EUCountryCode | undefined;
const location =
  countryCode === SupportedCountryCodes.EU
    ? (euCode ? t(euCode) : '')
    : userLocationDetails?.administrativeAreaLevel1 ?? '';
```